### PR TITLE
[SDK-673] iOS 14 locations docs updates

### PR DESCRIPTION
**Pull request by chshapiro**
https://jira.braze.com/browse/SDK-673

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/advanced_use_cases/locations_and_geofences.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ To support geofences for iOS:
 
 2. Braze Geofences [must be enabled][1] through the SDK either implicitly by enabling location collection or explicitly by enabling geofence collection. They are not enabled by default.
 
+__Note:__ Geofences will not work for users who give Approximate Location permission.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/advanced_use_cases/locations_and_geofences.md:17]:
```suggestion
{% alert important %}
As of iOS 14, Geofences does not work for users who choose to give their Approximate Location permission.
{% endalert %}
```

File: _docs/_developer_guide/platform_integration_guides/ios/analytics/location_tracking.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ page_order: 6
 
 By default, Braze disables location tracking. We enable location tracking after the host application has opted in to location tracking and gained permission from the user. Provided that users have opted into location tracking, Braze will log a single location for each user on session start.
 
+__Note:__ In order for location tracking to work in iOS 14 for users who give Approximate Location permission, you must update your SDK version to `3.26.1`.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/ios/analytics/location_tracking.md:11]:
```suggestion
{% alert important %}
In order for location tracking to work in iOS 14 for users who give Approximate Location permission, you must update your SDK version to at least `3.26.1`.
{% endalert %}
```

File: _docs/_developer_guide/platform_integration_guides/ios/analytics/location_tracking.md [hunk 3]
```diff
@@ -8,6 +8,10 @@ page_order: 6
 
 By default, Braze disables location tracking. We enable location tracking after the host application has opted in to location tracking and gained permission from the user. Provided that users have opted into location tracking, Braze will log a single location for each user on session start.
 
+{% alert important %}
+In order for location tracking to work in iOS 14 for users who give Approximate Location permission, you must update your SDK version to at least `3.26.1`.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/ios/analytics/location_tracking.md:12]: work -> work reliably

Comment (chshapiro) [reply _docs/_developer_guide/platform_integration_guides/ios/analytics/location_tracking.md:12]: updated

**Pull request merged**